jsonschema2pojo-core/src/main/java/org/jsonschema2pojo/util/URLUtil.java

```diff
@@ -21,12 +21,28 @@
 import java.net.URI;
 import java.net.URISyntaxException;
 import java.net.URL;
+import java.util.Arrays;
+import java.util.Set;
+import java.util.stream.Collectors;
 
 import org.apache.commons.lang.StringUtils;
 import org.jsonschema2pojo.URLProtocol;
 
 public class URLUtil {
 
+    private static final Set<URLProtocol> LOCAL_PROTOCOL = Arrays.stream(new URLProtocol[]{URLProtocol.NO_PROTOCOL, URLProtocol.FILE}).collect(Collectors.toSet());
```

Comment:
Can this just be replaced by `Set.of`?

Reply:
I'm afraid it can't since `Set::of` was introduced in Java 9 and current compiler configuration is:
https://github.com/joelittlejohn/jsonschema2pojo/blob/34cc69306fad2f5e27caa4c74cba9e16d8a0234a/pom.xml#L213-L223

That being said it could be changed to `= EnumSet.of(URLProtocol.NO_PROTOCOL, URLProtocol.FILE)` with a small caveat - resulting set would be mutable (however that shouldn't be a problem since it's _private_ and there are no methods returning it's reference.

Reply:
I changed the corresponding line. I am now Using an EnumSet wrapped in an unmuttable set.

Reply:
@joelittlejohn: Can it be merged now?